modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -335,7 +335,7 @@ export default class Ecdsa {
     return {
       i: oShare.i,
       y: oShare.y,
-      r: bigIntToBufferBE(r, 32).toString('hex'),
+      R: pointR.toHex(true),
```

Comment:
This might be a breaking change @olibeyene , you may need to address this in WP

Reply:
yes this will be braking change. can we keep R lower case?

Reply:
I think it's better that fields communicate their mathematical value. In this case, the value in the field used to be `r` which is the `x` coordinate of the "random" point used in signing. For calculating the `recid` value, we need the parity of the random point's `y` coordinate. So we essentially need the full random point `R` itself. Since we only need to know the `y`-parity, we just use the compressed value and interpret its `y`-parity from its SEC1 encoding prefix byte (`02` being an even parity, `03` being an odd parity).

Reply:
Since this is a breaking change, we need to reflect that in the commit message for the SDK. ref: https://github.com/BitGo/BitGoJS/pull/2565

We are also working on a PR template so that we can better communicate this effort: https://github.com/BitGo/BitGoJS/pull/2724